### PR TITLE
Create punisher.yml

### PR DESCRIPTION
**Pull request by superiorxman**
https://static.comicvine.com/uploads/scale_small/0/4/64741-11164-97738-1-punisher.jpg

Punishers arsenal

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/chars/punisher.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class: punisher
+costumes:
+- chatcolor: '&7'
+  name: §fDe§cad§8sh§co§8t
```
Comment (TheLightShadow) [inline CommandHelper/LocalPackages/commands/chars/punisher.yml]: Name should be changed to Punisher, not Deadshot.

File: CommandHelper/LocalPackages/commands/chars/punisher.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+effects: []
+items:
+- /shot give <plr> pistol
+- /shot give <plr> AK-47
+- /shot give <plr> Type95
+- /shot give <plr> Olympia
+- /shot give <plr> Python
+- /shot give <plr> Gauss
+- /shot give <plr> Grenade
+- /shot give <plr> Hunting
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/punisher.yml]: Use the same weapons as arkham knight, we aren't using "shot give" stuff anymore

Comment (superiorxman) [reply CommandHelper/LocalPackages/commands/chars/punisher.yml]: what do we use?

Comment (malonnnn) [reply CommandHelper/LocalPackages/commands/chars/punisher.yml]: Use the same weapons as arkham knight, we aren't using "shot give" stuff anymore

**Pull request merged**